project/Build.scala

```diff
@@ -814,22 +826,22 @@ object Build {
       },
 
       scriptedDependencies := {
-        scriptedDependencies.dependsOn(
-            publishLocal in compiler,
-            publishLocal in library,
-            publishLocal in testInterface,
-            publishLocal in testBridge,
-            publishLocal in jUnitPlugin,
-            publishLocal in jUnitRuntime,
-            publishLocal in irProject,
-            publishLocal in logging,
-            publishLocal in linkerInterface,
-            publishLocal in linker,
-            publishLocal in jsEnvs,
-            publishLocal in nodeJSEnv,
-            publishLocal in testAdapter,
-            publishLocal in jsEnvs,
-        ).value
+        //scriptedDependencies.dependsOn(
+        //    publishLocal in compiler,
+        //    publishLocal in library,
+        //    publishLocal in testInterface,
+        //    publishLocal in testBridge,
+        //    publishLocal in jUnitPlugin,
+        //    publishLocal in jUnitRuntime,
+        //    publishLocal in irProject,
+        //    publishLocal in logging,
+        //    publishLocal in linkerInterface,
+        //    publishLocal in linker,
+        //    publishLocal in jsEnvs,
+        //    publishLocal in nodeJSEnv,
+        //    publishLocal in testAdapter,
+        //    publishLocal in jsEnvs,
+        //).value
```

Comment:
Uncomment & fix.

Reply:
@sjrd, this is a problem: I forgot to uncomment this, but the relevant CI project passed anyways. Are we caching too much ivy stuff? (both scripted and our own home-brewn sbt test rely on `publishLocal`, so it must not be cached).

Reply:
Hum, I wouldn't be surprised if we did not delete the contents of the local Ivy repository between builds, indeed :s